migrations/169_grading_jobs__add_gradable_attribute.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE grading_jobs ADD COLUMN gradable BOOLEAN;
+UPDATE grading_jobs SET gradable=true;
```

Comment:
Do we need to set old grading jobs to have `gradable=true`? Maybe leaving them with `NULL` is clearer, because we don't actually know their status?